topbar.js

```diff
@@ -574,30 +574,35 @@ function disable() {
 }
 
 function fixTopBar() {
-    let spaces = Tiling.spaces
-    if (!spaces)
-        return;
-    let space = spaces.monitors.get(panelMonitor);
+    let space = Tiling.spaces?.monitors.get(panelMonitor) ?? false;
```

Comment:
Just to explain this line (in case others aren't use to such js operators) - it's using an optional chain with the nullish coalescing operator (??).

Basically, if `Tiling.spaces` is null (or undefined) then the nullish coalescing operator (??) means `space` will be assigned to `false`; otherwise it will return the left hand side, with optional `?.` function applied, meaning `space` will be assigned the value of `Tiling.spaces.monitors.get(panelMonitor)` (since `Tiling.spaces` is not `null` and not `undefined`).